src/main/java/ucb/internship/backend/models/InternshipMajor.java

```diff
@@ -54,11 +54,11 @@ public void setMajor(Major major) {
         this.major = major;
     }
 
-    public Integer getStatus() {
+    public boolean isStatus() {
```

Comment:
is Status doesn't make clear the intention, should be isActive or whatever you need